packages/dcos-integration-test/extra/test_3dt.py

```diff
@@ -283,6 +283,27 @@ def test_3dt_units(cluster):
         )
 
 
+@retrying.retry(wait_fixed=2000, stop_max_delay=LATENCY*1000)
```

Comment:
Seems to fire properly the first time: https://teamcity.mesosphere.io/viewLog.html?buildId=365550&buildTypeId=ClosedSource_Dcos_IntegrationTests_CloudIntegrationTests_DcosOssCcmIntegration_2&tab=testsInfo